[#67015] Documentation: Custom font for all PDF Exports (#20173)
opf/openproject: docs/system-admin-guide/design/README.md

sidebar_navigation:
  title: Design
  priority: 870
description: Custom color, theme, logo and PDF.
keywords: custom color, theme, logo, PDF
---
# Set custom color theme and logo (Enterprise add-on)

As an OpenProject Enterprise add-on you can replace the default OpenProject logo with your own logo. In addition, you can define your own color theme which allows you to implement your corporate identity in OpenProject.

Navigate to *Administration* → *Design* in order to customize your OpenProject theme and logo.

The design page provides several options to customize your OpenProject Enterprise edition, grouped in four tabs, **Interface, Branding, PDF export styles and PDF export font**. 

> You can [choose a color theme](#choose-a-color-theme) in the first two any of these tabs. 

Reviewer: ```suggestion
You can [choose a color theme](#choose-a-color-theme) in the first two tabs. 
```

Under **Interface** you can also choose [custom colors](#interface-colors) for elements of the interface, such as the primary button color, accent color, the background of the top navigation header and the main menu.

![Design interface settings in OpenProject administration](openproject_system_guide_design_interface.png)

Under the **Branding** tab you can also [upload a custom logo](#upload-a-custom-logo) to replace the default OpenProject logo, [set a custom favicon](#set-a-custom-favicon), which is shown as an icon in your browser window/tab, and [upload a custom touch icon](#set-a-custom-touch-icon), which is shown on your smartphone or tablet when you bookmark OpenProject on your home screen.

![Branding settings in OpenProject administration](openproject_system_guide_design_branding.png)

Under **PDF export settings** you can set the preferences
for e.g. [exporting work packages in a PDF format](../../user-guide/work-packages/exporting/#pdf-export),
meetings, timesheets… 

You can

> * upload a logo to be used in the head of an exported PDF

Reviewer: ```suggestion
* upload a logo to be used in the exported PDF header
```

* upload a custom background image for exports with a cover page
* adjust the text color of the text overlay at the custom background image of the cover page

> * upload a design image that is place left of the footer

Reviewer: ```suggestion
* upload a design image that is placed on the left side of the footer
```

> [!TIP]
> You can generate a demo PDF to see a preview of your settings. Click the **Generate Demo PDF** button.

> [!IMPORTANT]
> Only PNG or JPEG images can be used in PDF exports.

![PDF export styles settings in OpenProject administration](openproject_system_guide_design_pdf_export_styles.png)

> Under **PDF export font** you can upload a font family to be used in all PDF exports (e.g. Work packages report, Gantt, Meetings, Timesheet).

Reviewer: ```suggestion
Under **PDF export font** tab you can upload a font family to be used in all PDF exports (e.g. Work packages report, Gantt, Meetings, Timesheet).
```

> The font files must be in the TrueType Font (TTF) format. Maximal font file size is 40 MB.

Reviewer: ```suggestion
The font files must be in the TrueType Font (TTF) format. Maximum font file size is 40 MB.
```
